backend/processing/check_files_checksums/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt update && apt install -y python3
```

Comment:
We should [clean up the APT cache after installing](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) to minimize the size of the Docker layer.

Reply:
Done in de4abcf61c542be7bacf2e11517acde1a92063b2 . Thanks